packages/components/organisms/f-checkout/src/services/tests/addressService.test.js

```diff
@@ -174,4 +175,90 @@ describe('addressService', () => {
             });
         });
     });
+
+    describe('isAddressInLocalStorage ::', () => {
+        describe('if localStorage exists', () => {
+            beforeEach(() => {
+                Object.defineProperty(window, 'localStorage', { value: localStorageMock });
+            });
+
+            afterEach(() => {
+                window.localStorage.clear();
+                jest.resetAllMocks();
+            });
+
+            describe('when the address does NOT exist in local storage', () => {
+                it('should return false', () => {
+                    expect(addressService.isAddressInLocalStorage()).toBe(false);
+                });
+            });
+
+            describe('when the address does exist in local storage', () => {
+                it('should return true if there is a post code', async () => {
+                    window.localStorage.setItem('je-full-address-details', JSON.stringify(london3LinesDefault));
+                    expect(addressService.isAddressInLocalStorage()).toBe(true);
+                });
+
+                it('should return false if the customer only searched a postcode', async () => {
+                    window.localStorage.setItem('je-full-address-details', JSON.stringify({
+                        City: null,
```

Comment:
I think there should be a lat/long here too from what I remember?

Reply:
I believe that may be separate as i have another ticket for exactly that. I copied this straight from local storage 😄

Reply:
I actually think it's mapped to field 1 & 2 as you can't pass custom values but worth double checking.